activestorage/test/models/attached/one_test.rb

```diff
@@ -510,4 +510,11 @@ def avatar
       User.remove_method :avatar
     end
   end
+
+  test "loading associations after attaching" do
+    @user.avatar.attach create_blob(filename: "funky.jpg")
+
+    assert_not_nil @user.avatar_attachment
+    assert_not_nil @user.avatar_blob
```

Comment:
Ditto. Let’s fold these assertions into the first test, named “attaching an existing blob to an existing record.”